### DISCUSSION TITLE
Web content Whether the URL supports parsing the json data type？

### DISCUSSION THREAD

**Discussion by Blues-Code** [ramensoftware/windhawk-mods#274] [Q&A]
Hello, everyone
First of all, thank you @m417z for creating so many perfect mods for Windows.
I would like to ask how to configure the web url to get api style content.
Looking forward to your reply. Thank you！

**Comment by m417z** [comment]
Hi, I assume that you're referring to the [Taskbar Clock Customization](https://windhawk.net/mods/taskbar-clock-customization) mod. It only allows to match web content with a simple search, JSON isn't supported. Support can be added, but it's not trivial and I'm not sure how popular is the need for it, so I'll wait for more user feedback. Of course, you can fork the mod and add support yourself if you're familiar with C++.

**Reply by Blues-Code** [reply]
I am glad to receive your reply. Yes, it's the[Taskbar Clock Customization mod](https://windhawk.net/mods/taskbar-clock-customization)，that supports json data returned by HTTP restful, not just in XML format.Unfortunately, I don't know C++.
sincerely look forward to your support as soon as possible.
thanks again

**Reply by m417z** [reply]
Note that right now, XML isn't supported either. You can just search for the beginning and the end of the content. So in simple cases, it can work with JSON as well. For example, if you have:
```json
{
  "data": {
    "title": "This is a title"
  }
}
```
Then you can get the content between `"title": "` and `"`. It doesn't support all cases, for example if the title itself contains a quote, but it might be good enough.

**Reply by Blues-Code** [reply]
I know what to do. Thanks
